hazelcast/src/test/java/com/hazelcast/internal/ascii/HTTPCommunicator.java

```diff
@@ -113,13 +114,19 @@
     public static final String URI_LOG_LEVEL = "log-level";
     public static final String URI_LOG_LEVEL_RESET = "log-level/reset";
 
+    private static final String WWW_FORM_URL_ENCODED_MEDIA_TYPE = "application/x-www-form-urlencoded";
```

Comment:
this is the new default content type of communicator. as by default the `doPost()` use this encoding for POST body payload